File: 2578-split-with-minimum-sum/2578-split-with-minimum-sum.js
/**
 * @param {number} num
 * @return {number}
 */
var splitNum = function(num) {
    const temp=String(num).split("").sort((a,b)=>a-b)
    let num1=""
    let num2=""

    for(let i=0;i<temp.length;i++){
        if(i%2==0){
            num1+=temp[i]
        }else{
            num2+=temp[i]
        }
    }
    return Number(num1)+Number(num2)
};

File: 2578-split-with-minimum-sum/2578-split-with-minimum-sum.py
class Solution:
    def splitNum(self, num: int) -> int:
        temp=list(str(num))
        num1=""
        num2=""
        temp.sort()
        for i in range(len(temp)):
            if i%2==0:
                num1+=temp[i]
            else:
                num2+=temp[i]

        return int(num1)+int(num2)
        